master/buildbot/test/__init__.py

```diff
@@ -119,3 +119,6 @@
 
 # FIXME: needs to be sorted out (#3666)
 warnings.filterwarnings('ignore', "Invalid utf8 character string")
+
+# twisted.compat.execfile is using 'U' https://twistedmatrix.com/trac/ticket/9023
```

Comment:
I think it is important to keep the comment. the spellcheck linter must *not* encourage us to write less comments! @rutsky